about/reports/index.md

```diff
@@ -1,12 +1,12 @@
 # arXiv Reports
 
 ## Annual Reports
-[2020 Annual Report](https://indd.adobe.com/view/fdd63397-e4b0-41af-b479-4845dc1ef48e)
+[2020 Annual Report](https://indd.adobe.com/view/fdd63397-e4b0-41af-b479-4845dc1ef48e) ( [PDF version](2020_arXiv_annual_report.pdf) )
```

Comment:
Should we remove the InDesign link altogether, since it was causing some issues? 
```suggestion
[2020 Annual Report](2020_arXiv_annual_report.pdf)
```